### PR TITLE
feat: New quickstart for aws app signals

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New Quickstart based on AWS Application Signals. PR's are pending for entity's and documentation, but data is available today in AWS.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [ ] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [ ] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/amazon-cloudwatch-application-signals/config.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+icon: logo.png
+level: New Relic
+authors:
+  - New Relic
+title:  Amazon CloudWatch Application Signals
+documentation:
+  - name: Amazon CloudWatch Application Signals installation docs
+    description: |
+      Monitor AWS hosted services by monitoring with Amazon CloudWatch Application Signals
+    url: https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-cloudwatch-application-signals/
```
Comment (stevula) [inline quickstarts/aws/amazon-cloudwatch-application-signals/config.yml:28]: It looks like this link isn't working yet. I'm not sure we should release this with a dead link.
image

**Review by stevula — COMMENTED** [review]
Not including in this week's release due to inactive link (see comment above). Looks good otherwise!

**Review by sjyothi54 — APPROVED** [review]
Looks good

**Review by RamanaReddy8801 — APPROVED** [review]
LGTM

File: quickstarts/aws/amazon-cloudwatch-application-signals/config.yml (hunk 1 above)
Comment (sjyothi54) [reply quickstarts/aws/amazon-cloudwatch-application-signals/config.yml:28]: @stevula Merging PR since the doc url is working

**Pull request merged**